hiera.yaml

```
@@ -0,0 +1,16 @@
+---
+version: 5
+
+defaults:
+  datadir: 'data'
+  data_hash: 'yaml_data'
+
+hierarchy:
+  - name: 'Operating System Major Release'
+    path: '%{facts.os.family}/%{facts.os.release.major}.yaml'
+
+  - name: 'Operating System Family'
+    path: '%{facts.os.family}.yaml'
```

I prefer to remove ambiguity:
```suggestion
    path: 'family/%{facts.os.family}.yaml'
```

---

updated